packages/cli/src/databases/migrations/sqlite/1724951148974-AddApiKeysTable.ts

```diff
@@ -0,0 +1,61 @@
+import type { MigrationContext } from '@/databases/types';
```

Comment:
we need to update the sqlite driver before we can do this. [Typeorm currently recreates the table](https://github.com/n8n-io/typeorm/blob/master/src/driver/sqlite-abstract/AbstractSqliteQueryRunner.ts#L762-L787).

Reply:
@netroy out of curiosity, are we updating the driver anytime soon? 

@tomi I did like this because it's not possible to drop a column. I guess we ca do it in the same migration, and use the `isSqlite` flag (provided via context) and do something different, but that would be similar to what I have currently. The other approach is to do one migration that creates a temporary table, drop the old one, and then rename the temporary one, since that would work in the three databases. Not a fan of that, though.

Reply:
🤦 It is possible to drop, you just need to drop the unique index first. But yeah like @netroy mentioned would have to implement it in our typeorm fork first. So yeah let's go with this

Reply:
What approach exactly?  Using the `isSqlite` flag and moving what we have for SQLite to the common migration? Or renaming and dropping the table for the 3 databases (even though Postgres and MySQL do not need it)